### PR TITLE
[Filebeat] Remove experimental Citrix Netscaler and Symantec Endpoint modules

### LINKED ISSUE #23129: [Filebeat] Remove experimental Citrix Netscaler and Symantec Endpoint modules
The experimental Citrix Netscaler and Symantec Endpoint modules are not parsing logs as expected. They were created by converting Netwitness log parser definitions and didn't have test data. Let's remove them since they don't work as expected and hopefully we can provide replacements built on Ingest Node.

### PR DESCRIPTION
**Pull request by andrewkroh**
## What does this PR do?

The experimental Citrix Netscaler and Symantec Endpoint modules are not parsing logs as expected. They were created by converting Netwitness log parser definitions and didn't have test data. Let's remove them since they don't work as expected and hopefully we can provide replacements built on Ingest Node.

## Why is it important?

We want users to have a good experience with the modules we provide.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have made corresponding changes to the documentation
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Related issues

- Closes #23129

### CODE REVIEW

**Review by adriansr — APPROVED** [review]

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -947,6 +947,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Filebeat*
 
+- The experimental modules for Citrix Netscaler and Symantec Endpoint Protection
```
Comment (jamiehynds) [inline CHANGELOG.next.asciidoc:950]: The experimental modules for Citrix Netscaler and Symantec Endpoint Protection have been removed. As we continue to expand our coverage of common security data sources, we may consider supporting Citrix Netscaler and Symantec Endpoint Protection in a future release. 

@andrewkroh that sound ok?

Comment (andrewkroh) [reply CHANGELOG.next.asciidoc]: Incorporated. 👍

**Comment by andrewkroh** [comment]
run tests

**Comment by sbomma1973** [comment]
A customer is looking to make use of Citrix NetScaler Metrics and logs. What is the best way of collecting these logs.
@gose

**Comment by Ameurdx** [comment]
What is the best way to collect Citrix Netscaler logs?
[andrewkroh](https://github.com/andrewkroh)
[adriansr](https://github.com/adriansr)
Thank you

**Pull request merged**